var/spack/repos/builtin/packages/detray/package.py

```diff
@@ -77,6 +78,7 @@ class Detray(CMakePackage):
     depends_on("acts-algebra-plugins +vc", when="+vc")
     depends_on("acts-algebra-plugins +eigen", when="+eigen")
     depends_on("acts-algebra-plugins +smatrix", when="+smatrix")
+    depends_on("acts-algebra-plugins@0.26.0:", when="@0.87:")
```

Comment:
More an upstream comment, but where is this manifested in https://github.com/acts-project/detray/compare/v0.86.0...v0.87.0?